### PR TITLE
manual test for PaymentResponse.payerEmail attr

### PR DESCRIPTION
**Pull request by marcoscaceres**

### CODE REVIEW

**Comment by w3c-bots** [comment]
# Build PASSED

Started: 2017-08-21 05:55:23
Finished: 2017-08-21 06:02:07



View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/17587)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/266685156)

**Comment by w3c-bots** [comment]
# Build PASSED

Started: 2017-08-21 05:55:23
Finished: 2017-08-21 06:02:07



View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/17587)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/266685156)

**Comment by w3c-bots** [comment]
# Build PASSED

Started: 2017-08-21 05:55:23
Finished: 2017-08-21 06:02:07



View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/17587)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/266685156)

File: payment-request/payment-response/payerEmail-attribute-manual.https.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+<h2>payerEmail attribute</h2>
+<p>
+  When requested, please use "wpt@w3.org" as the email.
+</p>
+<ol>
+  <li>
+    <button onclick="runManualTest(this, { requestPayerEmail: undefined }, { payerEmail: null })">
+      payerEmail attribute is null when requestPayerEmail is undefined.
+    </button>
+  </li>
```
Comment (edenchuang) [inline payment-request/payment-response/payerEmail-attribute-manual.https.html]: Maybe we also can have a test for null PaymentOption, the default option set by browser can be tested. 
<button onclick="runManualTest(this, null, { payerEmail: null })">
  payerEmail attribute is null when PaymentOptions is set by browser.
</button>

**Review by edenchuang — APPROVED** [review]
LGTM

File: payment-request/payment-response/payerEmail-attribute-manual.https.html (hunk 1 above)
Comment (marcoscaceres) [inline payment-request/payment-response/payerEmail-attribute-manual.https.html]: Added as first test

**Review by edenchuang — APPROVED** [review]

**Pull request merged**